src/serialization/ttf2pkl.py

```diff
@@ -0,0 +1,155 @@
+from xml.etree import cElementTree
+from absl import flags, app
+import pickle
+import sys
```

Comment:
Is this file is the same as Jonny's in #40? If so, it doesn't need to be in this PR.

Reply:
Resolved.